index.html

```diff
@@ -133,7 +133,7 @@ <h3>The <dfn>isInputPending</dfn> Method</h3>
         </ol>
         <p class="note">The overall goal for this api is to allow user agents to return true if they think an event may end up getting dispatched to a browsing context soon, and not only if an event is already placed into the queue. This may mean that a user agent may look into the current configuration of iframes, other events in the queue, or other information when making decisions in step 4.1. For example, there may be cases where if a mousedown event is dispatched then a click event may be fired for this frame after the mousedown event. However if the mousedown event is canceled then the user agent knows that the click event will not be fired. In this case the user agent should return 'true' for a 'click' input before the canceling mousedown event, but after it knows that a click will not be fired it should return false.</p>
 
-        <p class="note">The specification for the isInputPending method is intentionally written in such a way that a user agent may return false for any reason. This is so user agents may include their own security measures while implementing this api and remain compliant with this specification.</p>
+        <p class="note">The specification for the isInputPending method is intentionally written in such a way that a user agent may return false for any reason. This is so user agents may include their own security and performance measures while implementing this api and remain compliant with this specification. For example, some user agents may not know which browsing context they would dispatch an event to without a prohibitively expensive computation, in that case it would be okay for the user agent to return false. It's never okay for a user agent to return true when it's unsure that the browsing context querying isInputPending is not the same one where the event will be dispatched.</p>
```

Comment:
Add note that the last sentence holds because it would be a violation of security, and otherwise would potentially allow untrusted iframes to observe input in other frames.